MAUI/DataForm/Editors.md

```diff
@@ -0,0 +1,1152 @@
+
+{% tabs %}
+{% highlight C# %}
+
+    [DataType(DataType.Date)]
+    [Display(Name ="Event Date")]
+    public DateTime? EventDate { get; set; }
+
+{% endhighlight %}
+{% endtabs %}
+
+
+#### Change the format of the date editor
+
+In the [DatePicker](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/datepicker?view=net-maui-7.0), short date will be shown by default. You can change the applied format by setting the `Format`property in `DataFormDateItem`.
+
+{% tabs %}
+{% highlight C# %}
+
+    dataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+
+    private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+    {   
+
+        if (e.DataFormItem != null && e.DataFormItem.FieldName == "Date" && e.DataFormItem is DataFormDateItem dateItem)
+        {
+            dateItem.Format = "dd, MM, yyyy";
+        }
+    }
+{% endhighlight %}
+{% endtabs %}
+
+`Image`
+
+#### Change the minimum and maximum date in date editor
+
+You can customize the maximum and minimum allowable dates in the [DatePicker](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/datepicker?view=net-maui-7.0) by setting `MaximumDate` and `MinimumDate` in the `DataFormDateItem` respectively.
+
+{% tabs %}
+{% highlight C# %}
+
+    dataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+
+    private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+    {
+        if (e.DataFormItem != null && e.DataFormItem.FieldName == "Date" && e.DataFormItem is DataFormDateItem dateItem)
+        {
+            dateItem.MinimumDate = new DateTime(2022, 5, 5);
+            dateItem.MaximumDate = new DateTime(2022, 9, 2);
+        }
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+`DateMAximumImages`
+
+## Time editor
+
+In the time editor, the [TimePicker](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/timepicker?view=net-maui-7.0&tabs=windows) will be loaded.
+
+
+In [TimePicker](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/timepicker?view=net-maui-7.0&tabs=windows), the default time value (12:00 AM) is displayed. You can also adding nullable `TimeSpan` data type for the time picker property in data form, which allows you to set default time and display the (12:00 AM) in time editor. 
+
+{% tabs %}
+{% highlight C# %}
+
+    [DataType(DataType.Time)]
+    [Display(Name = "Event Time")]
+    public TimeSpan? EventTime { get; set; }
+
+{% endhighlight %}
+{% endtabs %}
+
+
+#### Change the format of time editor**
+
+In the [TimePicker](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/timepicker?view=net-maui-7.0&tabs=windows), short time will be shown by default. You can change the applied format by setting the `Format`property in `DataFormTimeItem`.
```

Comment:
In the time editor, short time will be shown by default.

Reply:
content changed